### PR TITLE
Modiefied Color Theme

### PR DESCRIPTION
**Pull request by danwom**
### Summary of Changes
- Gray color scale shifted lighter, creates a softer feel
- Blue theme changed to cyan
Screen Shot 2019-05-03 at 2 11 52 PM

### Tests
Visual check
### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x ] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes, including screenshots of any UI changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public python functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes all tests documented in the [developer guide](https://github.com/lyft/amundsenfrontendlibrary/blob/master/docs/developer_guide.md#testing)
- [x] I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

File: amundsen_application/static/css/_variables-default.scss [hunk 1]
```diff
@@ -69,14 +71,13 @@ $btn-primary-color-hover:  white !default;
 
 $btn-default-bg:           white !default;
 $btn-default-bg-hover:     $gray-lightest;
-$btn-default-border:       $gray-lighter !default;
-$btn-default-border-hover: $gray-light !default;
-$btn-default-color:        $gray-dark !default;
-$btn-default-color-hover:  $gray-dark !default;
```
Comment (danwom) [inline amundsen_application/static/css/_variables-default.scss]: `$btn-default-color-hover` is not used in bootstrap or our code.

File: amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/styles.scss [hunk 2]
```diff
@@ -44,7 +44,7 @@
       margin-right: 8px;
     }
     .type {
-      color: $brand-color-3;
+      color: $brand-color-4;
```
Comment (danwom) [inline amundsen_application/static/js/components/TableDetail/DetailList/DetailListItem/styles.scss]: Needed to make this darker to meet contrast standards.

**Comment by danwom** [comment]
CC: @verdan

**Comment by verdan** [comment]
This is helpful. Thanks @danwom 👏

**Pull request merged**